data/pg/works/expense/ApplicableCharges.json

```diff
@@ -0,0 +1,75 @@
+{
+    "tenantId": "pg",
+    "moduleName": "calculator",
```

Comment:
I had checked with Aniket regarding this folder structure and module name, it works with this way but as a pattern we always follow using the module name as folder name which will easily identify the location of the file.